src/ModelLoaderAndroid.cpp

```diff
@@ -257,6 +258,7 @@ ModelLoaderAndroid::Run(void* data) {
   m.context->BindToThread();
   bool attached = false;
   if (m.jvm->AttachCurrentThread(&(m.env), nullptr) == 0) {
+    SetThreadName(+"VRB Loader");
```

Comment:
What the `+` for?

Reply:
I have no idea how that got in there. Thanks.

Reply:
Ok, merge the PR when that's fixed